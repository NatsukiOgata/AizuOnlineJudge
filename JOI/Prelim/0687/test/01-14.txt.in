70
527 581 358 380 547 590 499 509 457 479 600 439 477 483 508 487 531 494 482 460 592 363 361 542 405 404 410 589 396 419 526 541 449 382 556 594 389 560 426 403 503 440 462 468 444 565 548 543 475 367 545 532 524 510 466 407 446 550 549 540 408 505 378 359 511 502 504 517 501 538
