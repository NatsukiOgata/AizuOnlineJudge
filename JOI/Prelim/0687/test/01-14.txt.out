4927
28275
